vnext/src/Libraries/Components/View/ViewWindows.tsx

```diff
@@ -4,9 +4,16 @@
  * @format
  */
 
+import * as React from 'react';
 import {IViewWindowsProps} from './ViewWindowsProps';
-import React = require('react');
 
+/**
+ * Same as {@link https://facebook.github.io/react-native/docs/view | react-native's View}, but with extra windows specific functionality
+ *
+ * @remarks
+ * Prop type: {@link IViewWindowsProps}.
+ *
```

Comment:
Where is the schema documented?  We'll need to provide a link and guidance somewhere to reduce friction for devs.

Reply:
https://api-extractor.com/pages/tsdoc/doc_comment_syntax/